cmd/kaniko-docker/main.go

```diff
@@ -146,9 +163,26 @@ func run(c *cli.Context) error {
 			EnableCache:   c.Bool("enable-cache"),
 			CacheRepo:     c.String("cache-repo"),
 			CacheTTL:      c.Int("cache-ttl"),
+			DigestFile:    digestFileName,
 		},
 	}
-	return plugin.Exec()
+	err = plugin.Exec()
+	if err != nil {
+		return err
+	}
+
+	if c.String("output-file") != "" {
+		digestValue, err := digest.ReadDigestFile(digestFileName)
```

Comment:
read the file here only since it is just 3 lines of code.

Reply:
done